main_functional_test.go

```diff
@@ -216,6 +218,108 @@ var _ = Describe("Functional", func() {
 			})
 		})
 	})
+
+	Describe("AWS SQS", func() {
+
+		// Get secrets somehow
+		var queueName string
+
+		BeforeEach(func() {
```

Comment:
I think this BeforeEach and AfterEach has to be inside the Describe block further below - otherwise, you will have both test cases writing and reading from the same queue and potentially run into a race (causing the test to be flakey).

Reply:
Moved